src/controllers/profileController.js

```diff
@@ -0,0 +1,240 @@
+import model from '../../models'
+
+const { Profile } = model
+const { Employee } = model
+const { Role } = model
+
+ 
+
+class Profiles {
+  static create (req, res) {
+    const { telephone, firstname, lastname, address, dateOfBirth, gender } = req.body
```

Comment:
Why do you re-enter the firstname and lastname yet you already have these saved in Employee's DB. Why not just require them from the Employee's DB by including them. I think you should include in the body only new information and the rest since it is existing in the database you just require it since the two models already have a relation.